### PR TITLE
[Zh] sync 5 files in /security/authorization/authz-custom/

### PR DESCRIPTION
**Pull request by gssjl2008**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 1]
```diff
@@ -113,82 +249,88 @@ spec:
...
+
+### 网络负载均衡器 {#network}
+
+如果您正在使用保留客户端 IP 地址的 TCP/UDP 网络负载均衡器 (AWS 网络负载均衡器、
+GCP 外部网络负载均衡器、Azure 负载均衡器)，或者您正在使用轮询DNS，则您可以通过绕过
+kube-proxy 并阻止其将流量发送到其他节点，使用 `externalTrafficPolicy: Local`
+设置来同时保留 Kubernetes 内部的客户端 IP。
+
+{{< warning >}}
+For production deployments it is strongly recommended to **deploy an ingress gateway pod to multiple nodes** if you enable `externalTrafficPolicy: Local`. Otherwise, this creates a situation where **only** nodes with an active ingress gateway pod will be able to accept and distribute incoming NLB traffic to the rest of the cluster, creating potential ingress traffic bottlenecks and reduced internal load balancing capability, or even complete loss of ingress traffic to the cluster if the subset of nodes with ingress gateway pods go down. See [Source IP for Services with `Type=NodePort`](https://kubernetes.io/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport) for more information.
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:302]: Line 302 is required to localize too.

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 2]
```diff
@@ -113,82 +249,88 @@ spec:
...
+{{< /warning >}}
+
+Update the ingress gateway to set `externalTrafficPolicy: Local` to preserve the
+original client source IP on the ingress gateway using the following command:
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:306]: Lines 305-306 are required to localize too.

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 3]
```diff
@@ -197,45 +339,38 @@ spec:
...
-标头或代理协议来确定原始客户端 IP 地址，那么您应该在您的 `AuthorizationPolicy` 中使用
-`remoteIpBlocks`。如果您使用的是 `externalTrafficPolicy: Local`，那么您应该在
-`AuthorizationPolicy` 中使用 `ipBlocks`。
+**何时使用 `ipBlocks` 与 `emoteIpBlocks`:** 如果您使用 X-Forwarded-For HTTP 头部
+或代理协议来确定原始客户端 IP 地址，则应在 `AuthorizationPolicy` 中使用 `emoteIpBlocks`。
+如果您使用的是 `ExtraalTrafficPolicy：Local`，那么您的 `AuthorizationPolicy` 中应该使用
+`ipBlocks`。
 
-| 负载均衡器类型 | 客户端源 IP   | `ipBlocks` 与 `remoteIpBlocks`
+|Load Balancer Type |Source of Client IP   | `ipBlocks` vs. `remoteIpBlocks`
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:354]: Line 354 is required to localize too.

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 4]
```diff
@@ -286,22 +465,59 @@ EOF
...
+{{< text bash >}}
+$ CLIENT_IP=$(kubectl get pods -n foo -o name -l istio.io/gateway-name=httpbin-gateway | sed 's|pod/||' | while read -r pod; do kubectl logs "$pod" -n foo | grep remoteIP; done | tail -1 | awk -F, '{print $4}' | awk -F: '{print $2}' | sed 's/ //') && echo "$CLIENT_IP"
+192.168.10.15
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+* Update the `ingress-policy` to include your client IP address:
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:510]: Line 510 is required to localize too.

**Review by windsonsea — COMMENTED** [review]
Please provide a clear title to easily identify from your PR name.

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md (hunk 3 above)
Comment (gssjl2008) [reply content/zh/docs/tasks/security/authorization/authz-ingress/index.md:353]: Here, I don't see a problem.

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 5]
```diff
@@ -69,17 +112,78 @@ Kubernetes 的 `Ingress` 资源也必须由 Ingress 控制器支持，该控制
...
+
+|Cloud Provider | Load Balancer Name            | Load Balancer Type
+----------------|-------------------------------|-------------------
+|AWS EKS        | Classic Elastic Load Balancer | TCP Proxy
+|GCP GKE        | TCP/UDP Network Load Balancer | Network
+|Azure AKS      | Azure Load Balancer           | Network
+|IBM IKS/ROKS   | Network Load Balancer         | Network
+|DO DOKS        | Load Balancer                 | Network
+{{< tip >}}
+您可以指示 AWS EKS 在网关服务上创建带有注释的网络负载均衡器：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:132]:
```suggestion
您可以指示 AWS EKS 在网关服务上创建带有注解的的 Network Load Balancer：
```

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 6]
```diff
@@ -69,17 +112,78 @@ Kubernetes 的 `Ingress` 资源也必须由 Ingress 控制器支持，该控制
...
+
+{{< /tabset >}}
+
+{{< /tip >}}
+
+### TCP/UDP 代理负载均衡器 {#tcp-proxy}
+
+如果您使用的是 TCP/UDP 代理外部负载均衡器 (AWS Classic ELB)，
+它可以使用[代理协议](https://www.haproxy.com/blog/haproxy/proxy-protocol/)
+将原始客户端 IP 地址嵌入到分组数据中。外部负载均衡器和 Istio 入口网关都必须支持代理协议才能工作。
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:181]:
```suggestion
将原始客户端 IP 地址嵌入到分组数据中。外部负载均衡器和 Istio Ingress 网关都必须支持代理协议才能工作。
```

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 7]
```diff
@@ -113,82 +249,87 @@ spec:
...
+
+### 网络负载均衡器 {#network}
+
+如果您正在使用保留客户端 IP 地址的 TCP/UDP 网络负载均衡器 (AWS 网络负载均衡器、
+GCP 外部网络负载均衡器、Azure 负载均衡器)，或者您正在使用轮询DNS，则您可以通过绕过
+kube-proxy 并阻止其将流量发送到其他节点，使用 `externalTrafficPolicy: Local`
+设置来同时保留 Kubernetes 内部的客户端 IP。
+
+{{< warning >}}
+对于生产部署，如果您开启了 `exteralTrafficPolicy：Local`，强烈建议 **将一个入口网关实例部署到多个节点**。否则，这将导致 **只有** 具有活动入口网关实例的节点能够接受并将传入的 NLB 流量分发到集群的其余部分，从而造成潜在的入口流量瓶颈和降低的内部负载均衡能力，甚至在具有入口网关实例的节点子集关闭时完全丧失到集群的入口流量。有关更多信息，请参阅[服务源 IP `Type=NodePort`](https://kubernetes.io/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport)。
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:302]:
```suggestion
对于生产部署，如果您开启了 `externalTrafficPolicy: Local`，强烈建议 **将一个 Ingress 网关实例部署到多个节点**。否则，这将导致 **只有** 具有活动 Ingress 网关实例的节点能够接受并将传入的 NLB 流量分发到集群的其余部分，从而造成潜在的 Ingress 流量瓶颈和降低的内部负载均衡能力，甚至在具有 Ingress 网关实例的节点子集关闭时完全丧失到集群的 Ingress 流量。有关更多信息，请参阅[服务源 IP `Type=NodePort`](https://kubernetes.io/zh-cn/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport)。
```

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 8]
```diff
@@ -197,45 +338,38 @@ spec:
...
-标头或代理协议来确定原始客户端 IP 地址，那么您应该在您的 `AuthorizationPolicy` 中使用
-`remoteIpBlocks`。如果您使用的是 `externalTrafficPolicy: Local`，那么您应该在
-`AuthorizationPolicy` 中使用 `ipBlocks`。
+**何时使用 `ipBlocks` 与 `emoteIpBlocks`:** 如果您使用 X-Forwarded-For HTTP 头部
+或代理协议来确定原始客户端 IP 地址，则应在 `AuthorizationPolicy` 中使用 `emoteIpBlocks`。
+如果您使用的是 `ExtraalTrafficPolicy：Local`，那么您的 `AuthorizationPolicy` 中应该使用
+`ipBlocks`。
 
-| 负载均衡器类型 | 客户端源 IP   | `ipBlocks` 与 `remoteIpBlocks`
+|Load Balancer Type |Source of Client IP   | `ipBlocks` vs. `remoteIpBlocks`
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:353]:
```suggestion
| 负载均衡器类型 | 客户端源 IP   | `ipBlocks` 与 `remoteIpBlocks`
```
原文是正确的，无需修改

**Review by windsonsea — COMMENTED** [review]
以下词汇无需翻译，请全文检查一下：
- Envoy
- Ingress, ingress

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 9]
```diff
@@ -113,82 +249,87 @@ spec:
...
+如果您正在使用保留客户端 IP 地址的 TCP/UDP 网络负载均衡器 (AWS 网络负载均衡器、
+GCP 外部网络负载均衡器、Azure 负载均衡器)，或者您正在使用轮询DNS，则您可以通过绕过
+kube-proxy 并阻止其将流量发送到其他节点，使用 `externalTrafficPolicy: Local`
+设置来同时保留 Kubernetes 内部的客户端 IP。
+
+{{< warning >}}
+对于生产部署，如果您开启了 `externalTrafficPolicy: Local`，强烈建议 **将一个 Ingress 网关实例部署到多个节点**。否则，这将导致 **只有** 具有活动 Ingress 网关实例的节点能够接受并将传入的 NLB 流量分发到集群的其余部分，从而造成潜在的 Ingress 流量瓶颈和降低的内部负载均衡能力，甚至在具有 Ingress 网关实例的节点子集关闭时完全丧失到集群的 Ingress 流量。有关更多信息，请参阅[服务源 IP `Type=NodePort`](https://kubernetes.io/zh-cn/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport)。
+{{< /warning >}}
+
+使用以下命令更新入口网关以设置 `exteralTrafficPolicy：Local` 以保留入口网关上的原始客户端源IP:
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:305]:
```suggestion
使用以下命令更新 Ingress 网关以设置 `exteralTrafficPolicy：Local` 以保留 Ingress 网关上的原始客户端源 IP:
```

**Review by windsonsea — COMMENTED** [review]
请全文搜索：
- 入口
- 入口网关
找到并修改回 Ingress

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 10]
```diff
@@ -113,82 +249,87 @@ spec:
...
+
+### 网络负载均衡器 {#network}
+
+如果您正在使用保留客户端 IP 地址的 TCP/UDP 网络负载均衡器 (AWS 网络负载均衡器、
+GCP 外部网络负载均衡器、Azure 负载均衡器)，或者您正在使用轮询DNS，则您可以通过绕过
+kube-proxy 并阻止其将流量发送到其他节点，使用 `externalTrafficPolicy: Local`
+设置来同时保留 Kubernetes 内部的客户端 IP。
+
+{{< warning >}}
+对于生产部署，如果您开启了 `externalTrafficPolicy: Local`，强烈建议 **将一个 Ingress 网关实例部署到多个节点**。否则，这将导致 **只有** 具有活动 Ingress 网关实例的节点能够接受并将传入的 NLB 流量分发到集群的其余部分，从而造成潜在的 Ingress 流量瓶颈和降低的内部负载均衡能力，甚至在具有 Ingress 网关实例的节点子集关闭时完全丧失到集群的 Ingress 流量。有关更多信息，请参阅[服务源 IP `Type=NodePort`](https://kubernetes.io/zh-cn/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport)。
```
Comment (wilsonwu) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:302]:
```suggestion
对于生产部署，如果您开启了 `externalTrafficPolicy: Local`，
强烈建议 **将一个 Ingress 网关实例部署到多个节点**。
否则，这将导致 **只有** 具有活动 Ingress 网关实例的节点能够接受并将传入的 NLB 流量分发到集群的其余部分，
从而造成潜在的 Ingress 流量瓶颈和降低的内部负载均衡能力，
甚至在具有 Ingress 网关实例的节点子集关闭时完全丧失到集群的 Ingress 流量。
有关更多信息，请参阅[服务源 IP `Type=NodePort`](https://kubernetes.io/zh-cn/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport)。
```

**Comment by gssjl2008** [comment]
> 请全文搜索：
> 
> * 入口
> * 入口网关
> 
> 找到并修改回 `Ingress`

Has fixed

**Review by windsonsea — APPROVED** [review]
Thanks. It's time to merge though some minor errors remained there can be fixed later, which has little impact on the entire readability to readers.

**Pull request merged**